internal/provider/google/google_handler.go

```diff
@@ -61,13 +61,15 @@ func (provider *DNSProvider) updateIP(domain, subDomain, currentIP string) error
 	body, _ := io.ReadAll(resp.Body)
 	if resp.StatusCode != http.StatusOK {
 		log.Errorf("Update IP failed: %s", string(body))
-		return fmt.Errorf("update IP failed: %s", string(body))
+		return fmt.Errorf("Update IP failed: %s", string(body))
```

Comment:
In Go, error messages are usually written in lower-case.
Please refer to: https://earthly.dev/blog/golang-errors/